docs/articles/nunit/release-notes/framework.md

```diff
@@ -48,6 +48,7 @@ Total number of issues fixed in this release is : 105
 
 ### Bug fixes
 
+* [4532](https://github.com/nunit/nunit/issues/4532) Assert.That no longer allows 'null' message. Fixed by team [PR 4533](https://github.com/nunit/nunit/pull/4533)
```

Comment:
Not sure where to place this one. It is a bugfix on a bug introduced by a new feature. It is not a bugfix on 3.13.